tests/docker/test_dockercontainer_image_parsing.py

```diff
@@ -0,0 +1,12 @@
+from prefect.infrastructure.docker import DockerContainer
+
+def docker_image_parsing(image, tags):
+    d = DockerContainer(image=':'.join([image]+tags))
+    parsed_image, parsed_tag = d._get_image_and_tag()
+    assert parsed_image == image
+    assert parsed_tag == (tags[0] if len(tags)>0 else None)
+
+def test_dockercontainer_image_parsing():
```

Comment:
Could you use `pytest.parametrize` to test your image and tag combinations? We'll also want these tests to be in `tests/docker/test_parsing.py` for a parsing utility and `tests/infrastructure/test_docker_container.py` for anything that uses the `DockerContainer` type.

Reply:
I mode the requested changes, but saw no need to include a test specific for DockerContainer.